README.md

```diff
@@ -14,7 +14,73 @@ The exercise consists of three parts.
 
 ## SQL Challenge
 
-## Visualization
+A few things to note before you get started:
+
+* Please do all work for this portion of the exercise in SQL. While R or Python are useful tools, its important that you can use SQL to achieve the correct result. Any variant of SQL is fine but Postgres is preferred. 
+* There are blank/null values in the  data set and it reflects the nature of real world data.
+* The questions in this portion of the exercise are strictly to demonstrate SQL knowledge, so please take their intent with a grain of salt.
+* Use of documentation, online resources, or StackOverflow is _encouraged_. 
+
+### Instructions
+
+* Create a SQL database using the included [CSV files](/sql_challenge/data)
+* Use the database created to answer all the questions.
+* Provide all code you used to answer questions, set up the db, and import data in the [SQL Directory](/sql_challenge/sql)
+* Leave comments where you feel clarity is needed. 
+
+### Questions
+
+1. Data Integrity & Cleanup
+
+    Alphabetically list all the country codes in the continent map table that appear more than once. For countries with no country code make them display as "N/A" and display them first in the list. 
+
+2. List the Top 10 Countries by year over year % GDP per capita growth between 2011 & 2012. 
```

Comment:
I see blank gdp_per_capita for Countries in the data set.  Do you feel necessary to specifically mention how we expect them to handle null gdp_per_capita to report the Top 10?

Reply:
I'd like to see them make an informed decision on how to handle missing values